src/sentry/spans/consumers/process_segments/message.py

```diff
@@ -215,20 +233,16 @@ def transform_spans_to_event_dict(
         "type": "trace",
         "op": sentry_tags.get("transaction.op"),
         "span_id": segment_span["span_id"],
+        "hash": segment_span["hash"],
     }
 
     if (profile_id := segment_span.get("profile_id")) is not None:
         event["contexts"]["profile"] = {"profile_id": profile_id, "type": "profile"}
 
     for span in spans:
         event_span = cast(dict[str, Any], deepcopy(span))
-
-        if (op := span.get("sentry_tags", {}).get("op")) is not None:
-            event_span["op"] = op
```

Comment:
This is now part of enrichment, as the resulting attributes will be persisted into standalone spans.